connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/ArtifactSuite.scala

```diff
@@ -268,4 +269,14 @@ class ArtifactSuite extends ConnectFunSuite with BeforeAndAfterEach {
     val receivedRequests = service.getAndClearLatestAddArtifactRequests()
     assert(receivedRequests.size == 1)
   }
+
+  test("resolve ivy") {
+    val artifacts =
+      Artifact.newIvyArtifacts(URI.create("ivy://org.apache.hive:hive-storage-api:2.7.0"))
```

Comment:
Is there a risk that we hammer an external repo here?

Reply:
Probably, but many tests in `SparkContextSuite` also do this, though this is not a strong supporting argument 

I'm wondering if we should add `?repos=...`  parameter (or something similar) to the Ivy URI to support providing additional private / local repositories? Spark-core does support this by using Spark confs, but we can't do this in connect.

This will add some useful functionality and will allow us to test this without relying on remote repositories